docs/tutorials/optimizers_movingaverage.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Could you compare this against a more typical optimizer for comparison</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>